src/bentoml/testing/server.py

```diff
@@ -208,9 +211,10 @@ def run_bento_server_container(
     ]
     if config_file is not None:
         cmd.extend(["--env", "BENTOML_CONFIG=/home/bentoml/bentoml_config.yml"])
-        cmd.extend(
-            ["-v", f"{os.path.abspath(config_file)}:/home/bentoml/bentoml_config.yml"]
-        )
+        # NOTE: We don't set abspath path if running with bazel
+        if not run_in_bazel():
```

Comment:
Can we avoid this? Is this because we want to maintain two ways to run tests?

Reply:
I like just to do it one way with bazel.

Reply:
Yeah. Let's chat about this tmr.

Reply:
Yeah. The source code should not touch `pip install` things. This is an unnecessary coupling. 
Let build tools/ test tools to prepare the environment.

Reply:
there are some places that we need to change for bazel to work. Things like abspath or relpath wouldn't work inside bazel since it depends on symlink for incremental testing.